portfolio/src/main/java/com/google/sps/data/Post.java

```diff
@@ -0,0 +1,29 @@
+
+package com.google.sps.data;
+
+
+/**
+ * Class representing an individual post.
+ *
+ */
+public class Post {
+
+  private String title;
+
+  private String content;
+
+
```

Comment:
super nit.
1 blank line is good, and the two private class variables don't need a space between them

Reply:
Okay, fixed that.